client/src/components/User/registration.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+
+export const Registration = ({}) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confPassword, setConfPassword] = useState('');
+
+  const submit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <Row>
+      <Col xs={{ span: 2, offset: 5 }}>
+        <Card>
+          <Card.Header>Register</Card.Header>
+          <Card.Body>
+            <Form>
+              <Form.Group>
+                <Form.Label>Email</Form.Label>
+                <Form.Control type="emails" placeholder="Email" onChange={(e) => setEmail(e.target.value)} />
+              </Form.Group>
+              <Form.Group>
+                <Form.Label>Password</Form.Label>
+                <Form.Control type="password" placeholder="Password" onChange={(e) => setPassword(e.target.value)} />
+              </Form.Group>
+              <Form.Group>
+                <Form.Label>Repeat Password</Form.Label>
+                <Form.Control type="password" placeholder="Repeat Password" onChange={(e) => setConfPassword(e.target.value)} />
+              </Form.Group>
+              <Button variant="primary" type="submit" onClick={(e) => submit(e)}>
```

Comment:
can't you just do `={submit}` here?

Reply:
@alizand1992

Reply:
I think that would try to evaluate the variable submit which will be undefined